plugins/modules/mso_schema_template_vrf_rp.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Akini Ross (@akiniross) <akinross@cisco.com>
```

Comment:
```suggestion
# Copyright: (c) 2024, Akini Ross (@akiniross) <akinross@cisco.com>
```
Its been 2024 for a while now... 😆

Reply:
Changed, but a bit disappointed that you missed the one in the test file which also had comment mistakes. Please be a bit more thorough 😜.